sdk/textanalytics/Azure.AI.TextAnalytics/api/Azure.AI.TextAnalytics.netstandard2.0.cs

```diff
@@ -16,6 +16,12 @@ public partial class AnalyzeSentimentResultCollection : System.Collections.Objec
         public string ModelVersion { get { throw null; } }
         public Azure.AI.TextAnalytics.TextDocumentBatchStatistics Statistics { get { throw null; } }
     }
+    [System.FlagsAttribute]
+    public enum AnalyzeSentimentType
```

Comment:
I talked to Jonathan and show him the case. He doesn't like the Enum approach. We need to talk more about it.

Reply:
I understand that this doesn't work for Java. I'm curious to understand if his concerns were for .NET aswell

Reply:
Talked offline, and concerns are for Java because there is no Flags enum there.
Java will keep the original design of `IncludeOpinionMining` as a property